node/src/components/storage.rs

```diff
@@ -2113,7 +2113,7 @@ impl Storage {
         &self,
         deploy_hash: &DeployHash,
         finalized_approvals: &FinalizedApprovals,
-    ) -> Result<(), FatalStorageError> {
+    ) -> Result<bool, FatalStorageError> {
```

Comment:
Please consider extending the comment to indicate what's the meaning behind this `bool`.

Reply:
Done in [bc0e8db](https://github.com/casper-network/casper-node/pull/3441/commits/bc0e8db84799164fad1dbd9995d7bda793ff58b5).